server/routes/therapistRoute.js

```diff
@@ -3,13 +3,17 @@ const express = require('express')
 const router = express()
 
 // db models, if any
-const { Therapist, Patient } = require('../../db/models') // <-- CHANGE
+const Therapist = require('../../db/models/therapist')
 
 // -=-=-= CREATE =-=-=-
 
 // add something
 router.post('/', (req, res, next) => {
-
+  Therapist.create(req.body)
```

Comment:
I would prefer not to directly pass `req.body` as this lets any client place any fields/value pairs into the Sequelize create, we should just copy over key/values explicitly